README.md

```diff
@@ -47,23 +47,23 @@
 
 <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/code.png" width="400px" align="right">
 
-- *Machine learning on quantum hardware*. Connect to quantum hardware using **PyTorch**, **TensorFlow**, **JAX**, **Keras**, or **NumPy**. Build rich and flexible hybrid quantum-classical models.
+- <strong>*Program quantum computers*</strong>. Build quantum circuits with a wide range of state preparations, gates, and measurements. Run on [high-performance simulators](https://pennylane.ai/performance/) or [various hardware devices](https://pennylane.ai/plugins/), with advanced features like mid-circuit measurements and error mitigation.
 
-- *Just in time compilation*. Experimental support for just-in-time
+- <strong>*Machine learning with quantum hardware and simulators*</strong>. Integrate with **PyTorch**, **TensorFlow**, **JAX**, **Keras**, or **NumPy** to define and train hybrid models using quantum-aware optimizers and hardware-compatible gradients for advanced research tasks. [Quantum machine learning quickstart](https://docs.pennylane.ai/en/stable/introduction/interfaces.html).

```

Comment:
I would reverse the order between QML and Q algos and put quantum first. Our focus in PL is less on differentiable programming and more on meaningful algorithms

Reply:
Thanks @ixfoduap - this was updated in https://github.com/PennyLaneAI/pennylane/pull/6411/commits/c33e5911cd39d2e2c6d95fb2d4fb4bb585e214da . Let me know if that looks OK.